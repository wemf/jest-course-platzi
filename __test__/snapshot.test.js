import { getCharacter } from '../snapshot'
import rick from '../rick.json'

describe('Es hora de las instantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot()
    })
    /*
    test('Siempre fallar치 la instant치nea', () => {
        const user = {
            createdAt: new Date(),
            id: Math.floor(Math.random() * 20),
        }
        expect(user).toMatchSnapshot()
    })
    */

    test('Tenemos una excepci칩n dentro del c칩digo', () => {
        const user = {
            id: Math.floor(Math.random() * 20),
            name: "William Meza",
        }
        expect(user).toMatchSnapshot({
            id: expect.any(Number),
            name: "William Meza"
        })
    })
})