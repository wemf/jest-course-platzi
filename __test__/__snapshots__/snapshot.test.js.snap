// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las instantaneas Tenemos una excepción dentro del código 1`] = `
Object {
  "id": Any<Number>,
  "name": "William Meza",
}
`;
